pontoon/base/views.py

```diff
@@ -104,6 +105,17 @@ def locale(request, locale, template='locale.html'):
     return render(request, template, data)
 
 
+def locales(request, template='locales.html'):
```

Comment:
Why put the template in the view arguments? I can't imagine ever wanting a different template for this page.